### DISCUSSION TITLE
Mfx wet dry

### DISCUSSION THREAD

**Discussion by ptowns** [bangcorrupt/hacktribe#68] [Ideas]
Absolutely love the fact there is a mfx send. But a wet/dry would take it to the next level. My main isues are with applying reverb.

**Comment by bangcorrupt** [comment]
I think individual wet/dry per part would require modifying the sound engine.  It would be cool if we could hold the MFX button and set the mix with the volume knob.  I'd like to get hold of a mini Kaoss Pad 2 as I think they use the same FX engine with a different buss structure.  

Each MFX unit has a global wet/dry, for the reverbs this is controlled by the Y axis on the XY pad.  When I get as far as an FX preset editor, we'll be able to either set it at a fixed value, or map it to an axis on the pad.
